packages/o-spreadsheet-engine/src/functions/helpers.ts

```diff
@@ -541,7 +542,7 @@ export function applyVectorization(
 
   if (countVectorizedCol === 1 && countVectorizedRow === 1) {
     // either this function is not vectorized or it ends up with a 1x1 dimension
-    return formula(...args);
+    return formula.call(evalCtx, ...args);
```

Comment:
```suggestion
    return formula.apply(evalCtx, args);
```
to avoid spreading the `args`